packages/building-webpack/README.md

```diff
@@ -32,7 +32,7 @@ const createDefaultConfig = require('@open-wc/building-webpack/modern-and-legacy
 // import createDefaultConfig from '@open-wc/building-webpack/modern-config';
 
 module.exports = createDefaultConfig({
-  input: path.resolve(__dirname, './src/index.html'),
+  input: resolve(__dirname, './src/index.html'),
```

Comment:
I do think its path.resolve?

Reply:
Oh i see now. We should adjust the require, we never destructure it in any other examples.